glossary.md

```diff
@@ -6,28 +6,97 @@
 * `Library`: The collection of all `responses`.
 * `Registry`: The collection of available `survey objects` for use in `responses`.
 * `Analysis`: The process of converting many survey responses to a Sankey diagram by an
-  `Analyst` interacting with the `Library`. Surveys are searched by title and `tags`.
+  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or `object metadata`.
 
 
-## Response Object Registry
+## Registered Survey Object Registry
 
 A list of available `response objects`. Drives consistency in naming across `surveys` to
 support `analysis` and also to ease data entry for `Experts`, over time.
 
-Experts _may not_ add new `applications` to the `registry`, but _may_ add new `data
-products` and `observing systems` to the `registry`.
+Experts may add new `applications` `data products` and `observing systems` to the `registry`.
 
-Is suspect that we can just have a basic Registered Survey Object that has generic
+Is suspect that we can just have a basic `Registered Survey Object` that has generic
 fields whether it is a Data Product, Observing System, Etc. We probably need to have a
 separate structure to register Societal Benefit Analysis framework.
 
-_TODO_: Seek input from Bill Manley, ADC, Federated Search crew. Can we import or sync
-items in the registry from an external source?
+We want to align this data structure with best practices and partner organization
+structures (e.g. Polar Observing Assets working group, Arctic Data Center, Federated 
+Search crew). Evenually it's possible that we could import or sync items in the registry 
+from an external source.
 
 * `Name`
 * `Type`: `Observing system`, `data product`, `application`, `SBA`
 * _TODO_: Other fields?
 
+## Registered Survey Object Fields
+
+_NOTEfromHAZEL_ I think we can consolidate the `Observing System`, `Data Product`, 
+and `Application` sections below into this single list describing all survey objects, 
+except `Societal Benefit Areas`
```

Comment:
```suggestion


### Fields

```

I think this should be a subsection of the "Response object registry" section. These entities are intended to be re-usable classes of `Observing System`s, `Data Product`s, and `Application`s